addons/mail/tests/test_mail_thread.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.addons.mail.tests.common import MailCommon
+from odoo.tests import tagged
+
+
+@tagged('mail_thread', 'mail')
+class TestMailThread(MailCommon):
+
+    def test_search_message_partner_ids(self):
+        '''
+            The purpose of this test case is to check that _search_message_partner_ids does
+            not raise a traceback when called on a false domain
+        '''
+        message_partner = self.env['mail.thread']._search_message_partner_ids('in', [])
+        self.assertEqual([('id', 'in', [])], message_partner)
```

Comment:
Should check the result of a search directly, not the domain which could change for other reasons while still giving the right search results.

Also would be nice to test a few more cases to test the search generally, e.g.:
- search threads with no followers
- search thread with many followers using only one follower
- search threads with a Falsy domain
...

This way we cover it better and avoid other issues in the future, and it avoids having too specific tests all over the place.